server/server.go

```diff
@@ -43,6 +43,12 @@ type Config struct {
 type Server struct {
 	db    *db.DB
 	whois func(context.Context, string) (*apitype.WhoIsResponse, error)
+
+	// Metrics
+	countCalls             expvar.Map // :: method name → count
```

Comment:
use metrics.LabelMap instead, to produce prometheus friendly metric names

Reply:
Done! PTAL.